web-ui/src/app/pages/multi-step-page/steps.ts

```diff
@@ -122,11 +115,20 @@ class ParseStep extends Step<GlobalStateExampleBased> {
     async enterStep(state: GlobalStateExampleBased) {
         state.loading = true;
         state.currentStep = this;
+        state.valid = false;
+
+        await this.alpinoService.parseSentence(state.inputSentence)
```

Comment:
Op de een of andere manier komen we hier niet terecht als deze step actief is en je de pagina refreshed. Dan kan je naar de volgende stap zonder dat de voorbeeldzin geparsed is. Errors zijn ook niet meer zichtbaar?